with

```diff
@@ -20,33 +20,14 @@ print_options()
   echo "  -v, --version: Display the currently installed version of with"
 }
 
-print_help()
```

Comment:
Mostly its a matter of length. Things like customization and configuration that aren't command line arguments shouldn't be in the usage but instead should be documented somewhere else like the README.